<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Электрические схемы</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="zoomInPushButton">
        <property name="text">
         <string>Приблизить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomOutPushButton">
        <property name="text">
         <string>Отдалить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="3" colspan="2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="RenderArea" name="renderArea">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>627</width>
         <height>407</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="addElementPushButton">
        <property name="text">
         <string>Добавить элемент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="toggleElementPushButton">
        <property name="text">
         <string>Переключить элемент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeElementPushButton">
        <property name="text">
         <string>Удалить элемент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>Отменить действие</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="newFileAction"/>
    <addaction name="openFileAction"/>
    <addaction name="saveFileAction"/>
   </widget>
   <addaction name="fileMenu"/>
  </widget>
  <action name="newFileAction">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
  <action name="openFileAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header>RenderArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
